cclib/parser/orcaparser.py

```diff
@@ -1507,6 +1507,94 @@ def energy_intensity(line):
                 anisotropic.append(float(aniso))
 
             self.set_attribute('nmrtensors', nmrtensors)
+            
+        # -----------------------------------------------------------
+        #  NUCLEUS A = C    0 NUCLEUS B = C    1 
+        #  ( 13C  gnA =  1.405  13C  gnB =  1.405) r(AB) =     2.8677
```

Comment:
Related to your comment about isotope numbers, we may end up wanting the nuclear g-factors themselves, even if they are derived.  That's a "later" thing though.

Reply:
OK good to know. I ignored them because I couldn't decipher what gn was. This leans more towards having isotope numbers in the dict itself then